.github/workflows/ci-latest-release.yml

```diff
@@ -71,20 +71,35 @@ jobs:
           else
             echo "tag=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
           fi
-          RUNTIME=docker ./contribution/k3s/install_k3s.sh
+          RUNTIME=containerd ./contribution/k3s/install_k3s.sh
 
       - name: Generate KubeArmor artifacts
         run: GITHUB_SHA=$GITHUB_SHA ./KubeArmor/build/build_kubearmor.sh ${{ steps.vars.outputs.tag }}
 
-      - name: Deploy KubeArmor into Kubernetes
+      - name: Build Kubearmor-Operator
+        working-directory: pkg/KubeArmorOperator
         run: |
-          helm upgrade --install kubearmor ./deployments/helm/KubeArmor \
-          --values ./KubeArmor/build/kubearmor-helm-test-values.yaml \
-          --set kubearmor.image.tag=${{ steps.vars.outputs.tag }} \
-          --set kubearmorInit.image.tag=${{ steps.vars.outputs.tag }} \
-          -n kubearmor --create-namespace;
+          make docker-build
 
-          kubectl wait --for=condition=ready --timeout=5m -n kubearmor pod -l kubearmor-app
+      - name: deploy pre existing pod 
+        run: |
+          kubectl apply -f ./tests/k8s_env/ksp/pre-run-pod.yaml
+          sleep 60
+          kubectl get pods -A
+
+      - name: Run KubeArmor
+        run: |
+          docker save kubearmor/kubearmor-init:latest | sudo k3s ctr images import -
+          docker save kubearmor/kubearmor:latest | sudo k3s ctr images import -
+          docker save kubearmor/kubearmor-operator:latest | sudo k3s ctr images import -
+          docker save kubearmor/kubearmor-snitch:latest | sudo k3s ctr images import -
+        
```

Comment:
although I have updated, I'm thinking about what to do with the operator and snitch versions, can make them follow the same tag version here or build as latest, since they get pushed in another CI I guess.

Reply:
Folks, the `KubeArmorConfig` manifest that we have used only references `latest` images.
Ref - https://github.com/kubearmor/KubeArmor/blob/main/pkg/KubeArmorOperator/config/samples/kubearmor-test.yaml#L17-L22

So I think KubeArmor deployment will fail if images with the `latest` tag are not found. Let's export both tagged and `latest` images to k3s registry?

Reply:
We should be changing the manifest with required version right?

Reply:
Sure. The manifest is also references in ci-test-ginkgo.
Let's replace with sed?

Reply:
Sure sed works.